helpers/lido/deployment.ts

```diff
@@ -0,0 +1,81 @@
+import ethers from 'ethers';
+import {
+  AStETHFactory,
+  DefaultReserveInterestRateStrategyFactory,
+  StableDebtStETHFactory,
+  StETHMockedFactory,
+  VariableDebtStETHFactory,
+} from '../../types';
+import { strategySTETH } from '../../markets/aave/reservesConfigs';
+import { zeroAddress } from 'ethereumjs-util';
+import { Addresses } from './aave-mainnet-contracts';
+
+export async function deployStEthMock(deployer: ethers.Signer) {
+  return new StETHMockedFactory(deployer).deploy();
+}
+
+export async function deployAStETH(
+  lendingPoolAddress: string,
+  stEthAddress: string,
+  treasuryAddress: string,
+  deployer: ethers.Signer,
+  overrides?: ethers.Overrides
+) {
+  return new AStETHFactory(deployer).deploy(
+    lendingPoolAddress,
+    stEthAddress,
+    treasuryAddress,
+    'Aave interest bearing stETH',
+    `astETH`,
```

Comment:
The final tokens have different name and symbols. Could you please update their scripts to reflect the final tokens' configurations?